### ISSUE TITLE
transform rotateZ is not recognized

### ISSUE THREAD

**Issue by thasmo** [postcss/autoprefixer#537]
`transform: rotate(90deg)` does the same as `transform: rotateZ(90deg)` but autoprefixer doesn't recognize it.

**Comment by TrySound** [comment]
@thasmo Show input, output and browsers list.

**Comment by ai** [comment]
@thasmo as I understand `rotateZ` is not supported by IE 9, because `rotateZ` is part of 3D transforms, which are not supported by IE 9. So there is no need for `-ms-` prefix (it is necessary only for IE 9).

@thasmo @yisibl But maybe I am wrong. Can you test, does IE 9 support `rotateZ`.

**Comment by thasmo** [comment]
I thought about falling back to `rotate`, because, yes, IE 9 doesn't support `rotateZ`.
Input:
```css
body {
  transform: rotateZ(90deg);
}
```
Output:
```css
body {
  transform: rotateZ(90deg);
}
```
Expected Output:
```css
body {
  -ms-transform: rotate(90deg);
  transform: rotateZ(90deg);
}
```

**Comment by ai** [comment]
Hm. I can add this fix.

**Comment by thasmo** [comment]
It's just an idea, I have no clue if this has any side effects.

**Comment by ben-eb** [comment]
I am looking into writing a plugin that will minimise `rotateZ` to `rotate` and others, for [cssnano](https://github.com/ben-eb/cssnano), as the values are equivalent.

**Comment by ben-eb** [comment]
https://github.com/ben-eb/postcss-reduce-transforms will be included in cssnano 3.3.

**Comment by yisibl** [comment]
@ai Yes,  IE 9 doesn't support `rotateZ`.

**Comment by ai** [comment]
Done 6026227

**Comment by yisibl** [comment]
@thasmo What about `translateZ`?

**Comment by ai** [comment]
@yisibl `translateZ` is 3D command

**Comment by thasmo** [comment]
Ye, `translateZ` has no equivalent which works in IE9.

**Comment by thasmo** [comment]
Thanks for implementing btw! :)

**Comment by ai** [comment]
Released in 6.1
